<launch>

    <!-- Grasp generator node -->
    <node pkg="hb_grasp_generator" type="grasp_test" name="grasp_generator" output="screen" >
        <rosparam command="load" file="$(find hb_grasp_generator)/config/grasp_generator.yaml" />
    </node>

    <!-- RViz with shadow robot -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find hb_grasp_generator)/config/test.rviz" />


</launch>
